packages/web-components/src/components/rux-tabs/rux-tab/rux-tab.tsx

```diff
@@ -22,6 +23,7 @@ export class RuxTab {
         this.el.addEventListener('click', this._clickHandler)
 
         if (this.el?.parentElement?.getAttributeNode('small')) {
+            this.small = true
```

Comment:
discovered a separate issue with this where you can't toggle the small attribute

Reply:
yeah man who wrote this wtf